pangeo-notebook/packages.txt

```diff
@@ -109,78 +105,71 @@ docker-pycreds-0.4.0
 docopt-0.6.2
 docrep-0.2.7
 docutils-0.15.2
-eccodes-2.19.1
+eccodes-2.17.0
 entrypoints-0.3
 eofs-1.4.0
 erddapy-0.9.0
-esmf-8.0.1
-esmpy-8.0.1
+esmf-8.0.0
+esmpy-8.0.0
 expat-2.2.9
 fastapi-0.63.0
 fasteners-0.14.1
 fastjmd95-0.1
 fastprogress-1.0.0
 ffmpeg-4.3.1
-fiona-1.8.18
-font-ttf-dejavu-sans-mono-2.37
-font-ttf-inconsolata-2.001
-font-ttf-source-code-pro-2.030
-font-ttf-ubuntu-0.83
+fiona-1.8.13
 fontconfig-2.13.1
-fonts-conda-ecosystem-1
-fonts-conda-forge-1
 freetype-2.10.4
 freexl-1.0.5
 fribidi-1.0.10
 fsspec-0.8.5
 future-0.18.2
+g2clib-1.6.0
 gcc_impl_linux-64-9.3.0
 gcc_linux-64-9.3.0
 gcsfs-0.7.1
-gdal-3.1.4
-gdk-pixbuf-2.42.2
+gdal-3.0.4
 geocube-0.0.15
 geopandas-0.8.1
 geos-3.8.1
-geotiff-1.6.0
+geotiff-1.5.1
 geoviews-core-1.8.2
 gettext-0.19.8.1
 gflags-2.2.2
 giflib-5.2.1
-glib-2.66.4
+glib-2.66.3
 glog-0.4.0
 gmp-6.2.1
 gnutls-3.6.13
 google-auth-1.24.0
 google-auth-oauthlib-0.4.2
 graphite2-1.3.13
 graphviz-2.42.3
-grpc-cpp-1.34.1
+grpc-cpp-1.33.2
 gst-plugins-base-1.14.5
-gstreamer-1.18.3
+gstreamer-1.14.5
 gsw-3.4.0
-gtk2-2.24.33
-gts-0.7.6
 h11-0.12.0
 h5netcdf-0.8.1
-h5py-3.1.0
-harfbuzz-2.7.2
+h5py-2.10.0
```

Comment:
i'm worried that pynio is causing many other packages to drop major or minor versions (also gdal, cartopy, tiledb, scipy, etc...)

Reply:
yeah - maybe it's better to not merge it - i can try something else for now.